IdentityCore/src/oauth2/aad_base/MSIDAADTokenResponse.m

```diff
@@ -99,7 +105,9 @@ - (NSDictionary *)jsonDictionary
     NSMutableDictionary *json = [[super jsonDictionary] mutableDeepCopy];
     json[MSID_OAUTH2_CORRELATION_ID_RESPONSE] = self.correlationId;
     json[MSID_FAMILY_ID] = self.familyId;
+#if !EXCLUDE_FROM_MSALCPP
     json[MSID_TELEMETRY_KEY_SPE_INFO] = self.speInfo;
```

Comment:
Maybe you still want to get SPE info on CPP side?

Reply:
It's never called in MSAL CPP, and I don't see that will happen in the future.